app/src/main/AndroidManifest.xml

```diff
@@ -234,6 +234,10 @@
             android:name="com.duckduckgo.app.tabs.ui.TabSwitcherActivity"
             android:label="@string/tabActivityTitle" />
 
+        <activity
+            android:name="com.duckduckgo.app.tabs.ui.TabSwitcherBottomBarFeatureActivity"
```

Comment:
we create a new Activity here because it's easier to have it this way that change the current Activity. This will be either deleted (if the experiment fails) or become the new Tabs Activity (and remove it's counterpart)